src/wallet/spend.h

```diff
@@ -34,14 +34,38 @@ struct TxSize {
 TxSize CalculateMaximumSignedTxSize(const CTransaction& tx, const CWallet* wallet, const std::vector<CTxOut>& txouts, const CCoinControl* coin_control = nullptr);
 TxSize CalculateMaximumSignedTxSize(const CTransaction& tx, const CWallet* wallet, const CCoinControl* coin_control = nullptr) EXCLUSIVE_LOCKS_REQUIRED(wallet->cs_wallet);
 
+/**
+ * COutputs available for spending, stored by OutputType.
+ * This struct is really just a wrapper around OutputType vectors with a convenient
+ * method for concatenating and returning all COutputs as one vector.
+ *
+ * clear(), size() methods are implemented so that one can interact with
+ * the AvailableCoins struct as if it were a vector
+ */
 struct CoinsResult {
```

Comment:
`CoinsResult` was introduced in https://github.com/bitcoin/bitcoin/pull/25005 , which changed `AvailableCoins` to return a struct instead of populating a vector and was introduced here after a rebase.

Personally, I prefer `AvailableCoins` as the name of the struct (and renaming the `AvailableCoins` function to `GetAvailableCoins`), but changing the name again in this PR felt a little silly and would have expanded the diff on this PR. I have a follow-up PR in mind which incorporates some feedback from @furszy regarding the `CoinsResult` struct, and I think that would be a better place to address the naming